tools/browser-matcher/spec-parser.mjs

```diff
@@ -0,0 +1,33 @@
+const specSyntax = '{browserVersion}{operator:@|<|>|<=|>=}[/platformName]'
+export const SPEC_OPERATOR = {
+  AT: '@',
+  LT: '<',
+  GT: '>',
+  LTE: '<=',
+  GTE: '>='
+}
+
+/**
+ * Parses a browser spec string into it's constituent parts.
+ * @example
+ * // return { browserName: 'ie', operator: '@', browserVersion: 11', platformName: 'windows' }
+ * parseSpecString('ie@11/windows')
+ * @example
+ * // throws an error because `#` is not a valid operator
+ * parseSpecString('ie#11/windows')
+ * @param {string} spec A string representing a browser spec
+ * @returns {object} The parsed spec
+ */
+export function parseSpecString (spec) {
+  // Drop the platformName if it exists
+  const [browserInfo, platform] = spec.split('/')
+  const [specOperator] = browserInfo.match(/(@|<=|>=|<|>)/) || []
```

Comment:
In JIL, the spec string is actually like `safari@>=15.1,chrome@100,edge@<105`, etc, whereas the `@` operator here is intended to imply `==` rather than permanently existing as part of the spec.

I suppose it's fine to make it the users' (team) responsibility to know the new format, but also want to say this _could_ be hardier to account for JIL's format by checking for a 2nd operator.

Reply:
If we want to restore the syntax, I can. Right now, the string just needs an operator not an at sign and operator.
@metal-messiah @bjfield do you want me to restore the original syntax?

Reply:
I'm good either way